packages/visual-editor/src/hooks/useCanvasInteractions.ts

```diff
@@ -20,32 +20,36 @@ export default function useCanvasInteractions() {
       return;
     }
 
+    const [blockId, isAssembly] = draggableId.split(':');
```

Comment:
Please note that `isAssembly` is now a string `'false'` or `'true'` and thus always truthy. This will backfire in line 47.
In general, do you think there is a better approach compared to merging and again splitting this string which is supposed to be an "id".

Maybe, we can save us from all of this with the proposal I wrote in the [other PR](https://github.com/contentful/user_interface/pull/23249/files#r1753179850).